_gsdocs-proposals/2020/proposal_CVMFSPodmanIntegration.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Podman CVMFS integration
```

Comment:
You need to move this file from  _gsdocs-proposals/2020/ to _gsocproposals/2020/